backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java

```diff
@@ -49,7 +51,8 @@ public DocxGenerator(OutputFormatVariant outputFormatVariant, String description
     public byte[] generateOutputFile(Collection<LicenseInfoParsingResult> projectLicenseInfoResults, String projectName, String projectVersion, String licenseInfoHeaderText) throws SW360Exception {
         ByteArrayOutputStream docxOutputStream = new ByteArrayOutputStream();
         try {
-            XWPFDocument xwpfDocument = new XWPFDocument(this.getClass().getResourceAsStream(DOCX_TEMPLATE_FILE));
+            byte[] docxTemplateFile = CommonUtils.loadResource(DocxGenerator.class, DOCX_TEMPLATE_FILE).orElse(null);
```

Comment:
did you test what happens if `loadResource()` doesn't find the template? It looks like `orElse(null)` will cause an NPE at the next line. Maybe passing an empty byte array would be better. It would produce an empty output, but at least not fail outright. Logging a warning that the template hasn't been found would be beneficial, I imagine.

Reply:
corrected.